server/controllers/parties.js

```diff
@@ -1,97 +1,126 @@
-import partiesDb from '../models/parties';
+import { Pool } from 'pg';
+
+const pool = new Pool({
```

Comment:
You have made use of this pool in more that one place now, and its use is likely to increase. How about abstracting into a function.

Reply:
I first abstracted it based on the documentation on the node-postgres.com site and was calling it in the different pages but at some point, my code was not working properly and doing it this way was what made it to work

Reply:
Ok